lib/experian/precise_id/response.rb

```diff
@@ -57,32 +47,23 @@ def questions
       private
 
       def has_precise_id_section?
-        !!precise_id_server_section
+        !!precise_id_server
       end
 
-      def precise_id_server_section
-        hash_path(@response,"Products","PreciseIDServer")
+      def has_error_section?
+        super || !!precise_id_error_section
       end
 
-      def has_error_section?
-        !!error_section
+      def precise_id_error_section
```

Comment:
It would be great if we could document the response for future reference, like what nodes are present in the xml response? What do we consider a success, failure? etc. Nothing super formal it could be part of this repo in the README or an additional file now that the knowledge is fresh in your mind.